frontend/src/container/MetricsApplication/Tabs/DBCall.tsx

```diff
@@ -104,7 +104,7 @@ function DBCall(): JSX.Element {
 				>
 					View Traces
 				</Button>
-				<Card>
+				<Card data-testid="database_call_rps">
```

Comment:
would recommend to keep config of data test id against GraphTitle

Reply:
all the data-testid's are present against the attribute graph name

Reply:
Example: Graph -> name="database_call_rps"

Reply:
this is are hard coded magic string would recommend to keep this one in config as shared earlier to scale